db/pf-schema-X.Y.Z.sql

```diff
@@ -91,7 +91,8 @@ CREATE TABLE person (
   `custom_field_9` varchar(255) default NULL,
   `portal` varchar(255) default NULL,
   `source` varchar(255) default NULL,
-  PRIMARY KEY (`tenant_id`, `pid`),
+  `ipsk` varchar(255) NULL DEFAULT NULL,
```

Comment:
I wonder here if we should use ipsk or simply psk. If we do this for other vendors, "ipsk" as is wouldn't be right.